investment_tracking/views.py

```diff
@@ -18,14 +20,43 @@ class InvestmentTrackingSaveView(APIView):
     def post(self, request, startup_id):
         investor_profile = get_object_or_404(InvestorProfile, user=request.user)
         startup = get_object_or_404(StartUpProfile, id=startup_id)
+
         serializer = InvestmentTrackingSerializerCreate(data={'investor': investor_profile.id, 'startup': startup.id})
         if serializer.is_valid():
             try:
                 investment_tracking = InvestmentTracking(investor=investor_profile, startup=startup)
                 investment_tracking.save()
-                return Response({"message": "StartUp has been successfully saved."}, status = status.HTTP_201_CREATED)
+                return Response({"message": "StartUp has been successfully saved."}, status=status.HTTP_201_CREATED)
             except IntegrityError:
-                return Response({"message": "Startup is already saved."}, status = status.HTTP_400_BAD_REQUEST)
+                return Response({"message": "Startup is already saved."}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
In InvestmentTrackingSaveView, use status.HTTP_409_CONFLICT for duplicate save attempts instead of 400 Bad Request to specify the conflict.

Reply:
Done